spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -417,6 +418,34 @@ def expect_retryable_error(error_class)
           expect(submission.submitted_claim_id).to eq(Form526JobStatus.last.submission.submitted_claim_id)
         end
       end
+
+      context 'with toxic exposure Fipper enabled for user' do
```

Comment:
typo 🐬

Reply:
Corrected in [eb57d44](https://github.com/department-of-veterans-affairs/vets-api/pull/16802/commits/eb57d446d2e5536d9132817e6a34a9771910de87). Thanks!